README.md

```diff
@@ -42,10 +42,10 @@ You can get started testing on Google Colab or your local machine, by viewing ou
 
 ## Installation
 
-TensorTrade requires Python >= 3.7 for all functionality to work as expected.
+TensorTrade requires Python >= 3.6 for all functionality to work as expected.
 
 ```bash
-pip install -r requirements.txt
+pip install -r examples/requirements.txt

```

Comment:
This should be the base requirements.txt, not the examples one.

Reply:
Without the examples, the base implementation is worthless. I don't know for you but for me I need these requirements. The install used to refer to both requirements.txt files but the examples one already calls the first one. It was already like that but with an extra line. I just removed the unnecessary line.